ai-decision-services/ai-ad-dpp-engine/run-tests/run-tests.md

```diff
@@ -0,0 +1,280 @@
+Lab 8: A simple data processing, training and inferencing example
+===
+
+## Training
+
+To make sure you have everything setup well in training, you can upload the training data to the **training-data-bucket**. If you see a Data Flow run is triggered is successful, and under Anomaly Detection a model is trained successfully, then the training path is working.
+
+Fundamentally, you should see processed data in csv under your **staging-bucket**:
```

Comment:
suggestion: reword as "Once training completes, you should see processed data in csv in your **staging-bucket** under the folder specified in the config, in a subdirectory named ..."

Include the Spark format details here as well (part file and SUCCESS)

Reply:
Good point. Spark format detail is showing up in the screenshot.